test/gramework_test.go

```diff
@@ -64,21 +65,15 @@ func TestGrameworkHTTP(t *testing.T) {
 	app.SPAIndex("./nanotime.s")
 	app.GET("/sdnc_static/dist/*static", app.ServeDirNoCache("./"))
 	app.GET("/sdncc_static/dist/*static", app.ServeDirNoCacheCustom("./", 0, false, false, []string{}))
-	app.MethodNotAllowed(func(ctx *Context) {
+	app.MethodNotAllowed(func(ctx *gramework.Context) {
 		ctx.WriteString("GTFO")
 	})
 	app.UsePre(func() {
 		preCalled = true
 	})
```

Comment:
Error return value is not checked

Reply:
I need use this:
```go
if err != nil {
	t.Error(err.Error())
}
```
or something else?

Reply:
yes, but also add a log warn so we can understand what's going on